tests/tensor/test_nlinalg.py

```diff
@@ -555,14 +557,12 @@ def test_non_square_matrix(self):
 
     def test_negative_matrix(self):
```

Comment:
Couldn't this test be covered by simply adding a `-1` to the test parameters in `test_numpy_compare` above?

Reply:
Hmm, yeah actually make sense. Thanks!